scripts/testset_evaluation.py

```diff
@@ -0,0 +1,96 @@
+import os
+import argparse
+import json
+
+from fast_radiology.seed import random_seed
+from artificial_contrast.const import SEED
+
+random_seed(SEED)
+
+import torch
+import fastai
+import numpy as np
+import pandas as pd
+from fastai.vision import *
+from fastai.distributed import *
+
+from artificial_contrast.dicom import (
+    open_dcm_image_factory as simple_open_dcm_image_factory,
+    open_dcm_mask,
+)
+from artificial_contrast.data import get_scans, get_data, get_patients
+from artificial_contrast.learner import get_learner
+from artificial_contrast.const import (
+    FREQS_LIMIT_WINDOWS,
+    FREQS_NO_LIMIT_WINDOWS,
+    SIMPLE_MULTIPLE_WINDOWS,
+    SIMPLE_WINDOW_SMALL,
+)
+from artificial_contrast.evaluate import evaluate_patients
+
+
+# Config
+
+HOME_PATH = os.environ['HOME']
+DATA_PATH = os.environ['DATA']
+TEST_DATA_PATH = os.environ['TEST_DATA']
+MODEL_SAVE_PATH = os.environ['MODEL_SAVE']
+MODEL_NAME = os.environ['MODEL_NAME']
+DCM_CONF_PATH = os.environ['DCM_CONF']
+
+data_path = Path(DATA_PATH)
+test_data_path = Path(TEST_DATA_PATH)
+train_data_path = Path(TRA)
+
+EXPERIMENT_NAME = os.envrion['EXPERIMENT_NAME']
+
+IMG_SIZE = 512
+BS = 20
+
+
+DCM_LOAD_FUNC = {
```

Comment:
Maybe that would be extracted, or maybe we will only import the best method here and remove dict